### ISSUE TITLE
[Dropdown]: WCAG and other problems

### ISSUE THREAD

**Issue by ivarkallejarv** [TEHIK-EE/tedi-design-system#806]
## Problem

Might be an issue with dropdown or button or both

- Create a Dropdown using the button option, such as this:
```
            <Dropdown
              button={{ size: 'small', icon: 'add', children: 'Add', disabled: isPIA || !hasWriteAccess }}
              items={[{ children: 'Add X', onClick: () => doStuff() }]}
            />
```

Accessible text on such a button is empty
![Image](https://github.com/user-attachments/assets/e25d6378-732c-46fd-81fc-e2d13a4f059e)

**Comment by tobre6** [comment]
Community component.
Dropdown doesn't create aria tags. It's controlled via community button.
Should be fixed in community components.
